### PR TITLE
Wp 3557 form block

### PR DESCRIPTION
**Pull request by liktejas**

### CODE REVIEW

File: classes/class-responsive-block-editor-addons-frontend-styles.php [hunk 1]
```diff
@@ -14501,7 +14501,7 @@ public static function get_responsive_block_popup_css( $attr, $id ) {
 		}
 
 		/**
-		 * Get Defaults for Contact Form 7 Styler block
+		 * Get Defaults for Popup block.
```
Comment (pallavic27) [inline classes/class-responsive-block-editor-addons-frontend-styles.php:14504]: should be form block?

File: package-lock.json
Comment (pallavic27) [inline package-lock.json:1]: Is this file required?

File: package.json [hunk 2]
```diff
@@ -46,7 +48,7 @@
     "@babel/plugin-proposal-class-properties": "^7.14.5",
     "@testing-library/jest-dom": "^4.2.4",
     "@wordpress/block-editor": "3.1.0",
-    "@wordpress/block-library": "2.8.0",
+    "@wordpress/block-library": "^8.25.0",
```
Comment (pallavic27) [inline package.json:51]: Why this change?

File: classes/class-responsive-block-editor-addons-frontend-styles.php (hunk 1 above)
Comment (liktejas) [reply classes/class-responsive-block-editor-addons-frontend-styles.php:14504]: While making Popup Block, I forgot to change the description, So, I did it now.
The function is of popup block, not of form block.

File: package-lock.json
Comment (liktejas) [reply package-lock.json:1]: yes, because new packages have been installed, and this file will be used to install the packages with specific dependencies and their dependencies when anyone tries to install packages through the package.json

File: package.json (hunk 2 above)
Comment (liktejas) [reply package.json:51]: Updated to get all the new controls provided by WP Block Library
